galaxy_ng/tests/integration/api/test_ui_v2_teams.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+import pytest
+
+from galaxykit.client import BasicAuthClient
+from galaxykit.utils import GalaxyClientError
+
+from ..utils.namespaces import generate_namespace
+
+
+@pytest.mark.parametrize("endpoint", ["groups", "teams"])
+@pytest.mark.deployment_standalone
+@pytest.mark.min_hub_version("4.11.0dev")
+def test_ui_v2_teams(galaxy_client, endpoint):
+
+    gc = galaxy_client("admin", ignore_cache=True)
+    ga = BasicAuthClient(gc.galaxy_root, gc.username, gc.password)
+
+    # make the team via the _ui/v2/teams/ endpoint
+    random_name = generate_namespace()
+    endpoint_data = ga.post(
```

Comment:
I'm suprised this doesn't need to be skipped

Reply:
It should be skipped too, I'm going to skip it as it will (should) fail in the future if we do not.